CMakeLists.txt

```diff
@@ -27,8 +21,19 @@ set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
 set(CMAKE_INSTALL_RPATH $ORIGIN)
 
 # Project
-add_executable("${PROJECT_NAME}" "src/main.cpp" "src/pathtools_excerpt.cpp" "src/setup.cpp")
-target_link_libraries("${PROJECT_NAME}" PRIVATE "${OPENVR_LIB}" fmt::fmt-header-only ${CURSES_LIBRARIES})
+# Set some Win32 Specific Settings
+IF(WIN32)
+SET(GUI_TYPE WIN32)
+ENDIF(WIN32)
+# Set some Apple MacOS Specific settings
+IF (APPLE)
+SET(GUI_TYPE MACOSX_BUNDLE)
+ENDIF (APPLE)
+
+add_subdirectory(traypp EXCLUDE_FROM_ALL)
+link_libraries(tray)
```

Comment:
```suggestion
link_libraries(tray)

find_package(Threads REQUIRED)
```

Reply:
Under some system (linux), threads require a distinct library. So it is more portable to include threads explicitly.